taiga_halo2/examples/taiga_sudoku/main.rs

```diff
@@ -0,0 +1,8 @@
+pub mod app_vp;
+pub mod dealer_intent_vp;
+pub mod gadgets;
+pub mod intent_app_vp;
+use crate::app_vp::halo2_sudoku_app_vp_circuit_final;
+fn main() {
+    halo2_sudoku_app_vp_circuit_final();
```

Comment:
We may need a sudoku final transaction here later.
Let's merge it first.

Reply:
Which branch do you want to merge first?

Reply:
Merge this PR to xuyang/sudoku_example branch first
Then I'll refine the `intent_app_vp` and `dealer_intent_vp` in the branch xuyang/sudoku_example

Reply:
Perfect